src/main/java/net/dv8tion/jda/api/entities/Entitlement.java

```diff
@@ -0,0 +1,99 @@
+package net.dv8tion.jda.api.entities;
+
+
+import javax.annotation.Nullable;
+import java.time.OffsetDateTime;
+
+/**
+ * Represents a Discord {@link Entitlement Entitlement} for premium App subscriptions.
+ * <br>This should contain all information provided from Discord about an {@link Entitlement Entitlement}.
+ *
+ * @author Giulio Pimenoff V.
+ */
+public interface Entitlement
+{
+    /**
+     *
+     * @return The id of the {@link Entitlement Entitlement}
+     */
+    String getId();
+
+    /**
+     *
+     * @return The id of the SKU related to this {@link Entitlement Entitlement}
+     */
+    String getSkuId();
```

Comment:
All snowflake IDs should have a corresponding `getXIdLong()` method that returns the raw `long` instead of a string.

All methods must have `@Nonnull` or `@Nullable` if they return a reference type like `String`.